src/ua_connection.c

```diff
@@ -102,6 +108,8 @@ separateIncompleteChunk(UA_Connection *connection, UA_ByteString * UA_RESTRICT m
                 return retval;
             memcpy(connection->incompleteMessage.data, message->data, message->length);
             connection->releaseRecvBuffer(connection, message);
+            //The complete message is buffered, so no more content to be processed in message
+            message->length = 0;
```

Comment:
Isn't this redundant, since releaseRecvBuffer is called on the message? Internally this should free the message and set the length to 0, right? (ua_network_tcp just calls the deleteMembers function of UA_ByteString in its implementation)